fastlane/lib/fastlane/plugins/template/%gem_name%.gemspec.erb

```diff
@@ -13,9 +13,8 @@ Gem::Specification.new do |spec|
   spec.license       = "MIT"
 
   spec.files         = Dir["lib/**/*"] + %w(README.md LICENSE)
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
-
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
Side note that setting this to true will prevent us from pushing new versions of _fastlane_ to RubyGems using a CI job — which is something that I think we hope to achieve at some point, as discussed internally — instead of relying on @joshdholtz to do manual releases (and having him enter MFA codes during those) like now.

Obviously since @joshdholtz still haven't found time yet to work on sharing the permissions and automation for doing new releases, this isn't really a problem yet—though that still means that @joshdholtz  will have to configure MFA on his RubyGems account to still be able to do new releases—but if and when we end up having a system where multiple folks in the fastlane core team would be able to release a new version, we might be needing to disable that setting then.

All this means that maybe we don't want to turn this setting on (and instead disable the corresponding rule in rubocop)?

Reply:
Ah, fair point 😢 I guess I'll have to revert this until we have that CI automation in place... And yes, I added it because rubocop warned me 😅 I'll silent it for now, good catch!

Reply:
On a second thought, this file is just an erb template file used to generate plugins. It's not fastlane's gemspec file. AFAIK it's only used when a user generates a new fastlane plugin from the template. You sure this would still cause issues? 😬  @AliSoftware @getaaron

Reply:
Ah, good catch, yes, since it's only the `erb` template for plugins, then it makes sense to keep it for fastlane users who create their own plugins.

And I looked at [our own `.gemspec`](https://github.com/fastlane/fastlane/blob/6e20b52d2281226aae9888a7b5d3c01843c306b9/fastlane.gemspec) and it seems that MFA is not enabled in that one, so I think we're all good!

Reply:
Sweet!